### PR TITLE
[doc](random_sink) Add some doc content about random sink

### PR DESCRIPTION
**Pull request by caiconghui**
# Proposed changes

1. Add some doc content about random sink
2. Fix bug of showing missing rowsets info

## Problem summary

Describe your changes.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [ ] No
    - [ ] I don't know
3. Has unit tests been added:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
4. Has document been added or modified:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
5. Does it need to update dependencies:
    - [ ] Yes
    - [ ] No
6. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [ ] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 37.77 seconds
 load time: 563 seconds
 storage size: 17154887249 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20221022205156_clickbench_pr_32760.html

File: docs/zh-CN/docs/data-table/data-partition.md [hunk 1]
```diff
@@ -338,12 +338,18 @@ Doris 支持两层的数据划分。第一层是 Partition，支持 Range 和 Li
 
    > 注：表的数据量可以通过 [`SHOW DATA`](../sql-manual/sql-reference/Show-Statements/SHOW-DATA.md) 命令查看，结果除以副本数，即表的数据量。
 
+4. **关于 Random Distribution 的设置以及使用场景。**   
+    - 如果 OLAP 表没有更新类型的字段，将表的数据分桶模式设置为 RANDOM，则可以避免严重的数据倾斜。
```
Comment (morningman) [inline docs/zh-CN/docs/data-table/data-partition.md:342]: 这里加一下 random distribution 的原理说明吧。比如设置random 后，数据是“如何”被指定到“哪一个”tablet的。

**Review by morningman — APPROVED** [review]
LGTM

**Review by hf200012 — APPROVED** [review]
LGTM

**Pull request merged**